app/javascript/application.js

```diff
@@ -1,3 +1,8 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+$(".mul-select").select({
+   placeholder: "select Option",
+   tags: true,
+ });
+
```

Comment:
should be in dom ready, or else it may not work.

Reply:
It work.